eZ/Publish/Core/REST/Server/Controller/Content.php

```diff
@@ -473,8 +470,6 @@ public function updateVersion($contentId, $versionNumber, Request $request)
 
         try {
             $this->repository->getContentService()->updateContent($versionInfo, $contentUpdateStruct);
-        } catch (ContentValidationException $e) {
-            throw new BadRequestException($e->getMessage());
         } catch (ContentFieldValidationException $e) {
             throw new BadRequestException($e->getMessage());
```

Comment:
clearly, the errors need to be structured in the REST response (there's a proposal in the issue https://jira.ez.no/browse/EZP-24853)